src/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -1084,6 +1100,45 @@ public void sendBattleRecordsToGameData(final IDelegateBridge aBridge) {
     }
   }
 
+
+  /**
+   * 'Auto-fight' all of the air battles and strategic bombing runs.
+   * Auto fight means we automatically begin the fight without user action. This is to avoid clicks during the
+   * air battle and SBR phase, and to enforce game rules that these phases are fought first before any other combat.
+   */
+  void fightAirCombats(final IDelegateBridge delegateBridge) {
```

Comment:
Makes sense, 'fightBombingBattles' is a good suggestion, updated the method name to that.